code_generator/GdkCodeGenerator/Templates/UnityGameObjectComponentDispatcherGenerator.tt

```diff
@@ -200,6 +208,35 @@ namespace <#= qualifiedNamespace #>
 <# } #>
             }
 
+            public override void InvokeOnCommandResponseCallbacks(Dictionary<int, InjectableStore> entityIndexToInjectableStore)
+            {
+<# for (var j = 0; j < commandDetailsList.Count; j++) { #>
+
+                if (!CommandResponsesComponentGroups[<#= j #>].IsEmptyIgnoreFilter)
+                {
+                    var entities = CommandResponsesComponentGroups[<#= j #>].GetEntityArray();
+                    var commandResponseLists = CommandResponsesComponentGroups[<#= j #>].GetComponentDataArray<CommandResponses.<#= commandDetailsList[j].CommandName #>>();
+                    for (var i = 0; i < entities.Length; i++)
+                    {
+                        var injectableStore = entityIndexToInjectableStore[entities[i].Index];
+                        if (!injectableStore.TryGetInjectablesForComponent(commandResponseHandlerInjectableId, out var commandResponseHandlers))
+                        {
+                            continue;
+                        }
+
+                        var commandResponseList = commandResponseLists[i];
+                        foreach (Requirables.CommandResponseHandler commandResponseHandler in commandResponseHandlers)
+                        {
+                            foreach (var commandResponse in commandResponseList.Responses)
```

Comment:
cache `commandResponseList.Responses` to avoid calling the getter multiple times

Reply:
Won't it only call once to fetch the iterator?